invenio_app_ils/ui/src/invenio_app_ils/common/api/eitems/serializer.js

```diff
@@ -0,0 +1,23 @@
+import _isEmpty from 'lodash/isEmpty';
+import { fromISO } from '../date';
+
+export function serializeResponse(hit) {
+  let result = {};
+  if (!_isEmpty(hit)) {
+    result['id'] = hit.id;
+    result['created'] = fromISO(hit.created);
+    result['updated'] = fromISO(hit.updated);
+    if (hit.links) {
+      result['links'] = hit.links;
```

Comment:
Is this going to replace `{links: { self: {} }}`? Instead of replacing it, maybe we can do something like 
`result['links']['hits'] = hit.links;`

Reply:
No it should take the links as they come from the response!